ui-kit/PrayerCard/PrayerCard.stories.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Box } from 'ui-kit';
+
+import PrayerCard from './PrayerCard';
+import { CardCarousel } from '../';
+
+export default {
+  title: 'ui-kit/PrayerCard',
+  component: PrayerCard,
+};
+
+const cardProps = {
+  date: 'Aug 23, 2020',
+  description: 'This is where my prayer would be displayed.',
+  coverImage: 'https://source.unsplash.com/random/1000x1000',
+  mx: 's',
+};
+
+export const Default = () => (
+  <CardCarousel cardsDisplayed={4} hideArrows>
```

Comment:
I appreciate that you make the effort to keep up with the Storybook stories — good habit that I should be working on, too.